predictors.go

```diff
@@ -0,0 +1,64 @@
+package hermit
```

Comment:
Wouldn't it be better if this was under `shell`?

Reply:
Unfortunately, that would create a package cycle as we need to use `Env` here to interact with the environment. I thought it would be easier to keep this in the "top" package for that reason.